java/openjdk10/Portfile

```diff
@@ -57,7 +58,7 @@ destroot.violate_mtree yes
 destroot {
     set target ${destroot}/Library/Java/JavaVirtualMachines
     xinstall -m 755 -d ${target}
-    copy ${worksrcpath} ${target}
+    copy ${worksrcpath} ${target}/openjdk${openjdk_version}
```

Comment:
Shouldn't the `target` variable get this change instead? Otherwise the `xinstall` command does not act on it.

Reply:
@mf2k Wait no, that doesn't yield the same result. When changing that OpenJDK 11 ends up under `/Library/Java/JavaVirtualMachines/openjdk11/jdk-11.jdk`, but that's not correct. It should be `/Library/Java/JavaVirtualMachines/openjdk11`, which the original version of my pull request resulted in. I'll revert it.